test/README.md

```diff
@@ -12,7 +12,8 @@ To run only the integration tests, run from the test directory:
 
 Name                            | Default                   | Description
 :--                             | :---                      | :----------
-LXD\_BACKEND                    | dir                       | What backend to test against (btrfs, dir, lvm, zfs, or random)
+LXD\_BACKEND                    | dir                       | What backend to test against (btrfs, ceph, dir, lvm, zfs, or random)
+LXD\_CEPH\_CLUSTER              | ceph                      | The name of the ceph cluster to create osd pools in. Only used if "\${LXD_BACKEND}" = "ceph".
```

Comment:
My idea here is, if you set `LXD_BACKEND=ceph` you also need to set `LXD_CEPH_CLUSTER`. If you don't the test suite will give you an error about `LXD_CEPH_CLUSTER` not being set and that it's required. This also works in the random case as I added the following logic:
- if you set `LXD_BACKEND=random` and you do not set `LXD_CEPH_CLUSTER` then `ceph` will not be listed as an available backend even if the tools for it are available.
- if you set `LXD_BACKEND=random` and you set `LXD_CEPH_CLUSTER=<something>` then `ceph` is available as a backend.